exercises/leap/src/example.c

```diff
@@ -5,10 +5,10 @@ bool is_leap_year(int year)
    if (year % 4 == 0) {
 
       if (year % 400 == 0) {
-         return true;
+	 return true;
```

Comment:
I am struggling to understand how this could be indented only 2 spaces from the `if`. My current attempt at fixing things is very clearly wrong.